### ISSUE TITLE
Focus indicator doesn't meet 3:1 contrast ratio

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7471]
To meet the following WCAG rule: https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance the focus indicator needs to have a 3:1 contrast ratio. Currently the focus indicator color is not meeting this requirement. We should explore alternative colors.

**Comment by jcfilben** [comment]
Linking this issue to https://github.com/grommet/hpe-design-system/issues/4434
